src/client/hud.cpp

```diff
@@ -278,6 +278,25 @@ void Hud::drawItems(v2s32 upperleftpos, v2s32 screen_offset, s32 itemcount,
 	}
 }
 
+// Calculates screen position of waypoint. Returns true if waypoint is behind player (and thus not visible).
+bool Hud::calculateScreenPos(const v3s16 &camera_offset, HudElement *e, v2s32 *pos)
```

Comment:
Please invert this, it logically makes more sense for `false` to mean fail and `true` to mean success

Reply:
Well but it complicates the code... Will do so anyways...